test/multivariate/mvlognormal.jl

```diff
@@ -19,7 +19,7 @@ function test_mvlognormal(g::MvLogNormal, n_tsamples::Int=10^6,
     @test partype(g) == Float64
     @test isa(mn, Vector{Float64})
     if g.normal.μ isa Zeros{Float64,1}
-        @test md isa Fill{Float64,1}
+        @test md isa Union{Fill{Float64,1}, Ones{Float64,1}}
```

Comment:
Maybe we could even just relax it to
```suggestion
        @test md isa FillArrays.AbstractFill{Float64,1}
```
? I guess the main point of the test was to ensure that we do not return a dense vector.

Reply:
Yes, this should work as well. I just wanted to be more explicit.

Reply:
I think `AbstractFill` is fine here, and also relies a bit less on the broadcasting details in FillArrays.